internal/services/network/network_manager_verifier_workspace_reachability_analysis_intent_resource.go

```diff
@@ -0,0 +1,300 @@
+package network
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"time"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-helpers/lang/response"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonids"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonschema"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/cosmosdb/2024-08-15/cosmosdb"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/network/2024-05-01/reachabilityanalysisintent"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/network/2024-05-01/reachabilityanalysisintents"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/network/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/validation"
+)
+
+var _ sdk.Resource = ManagerVerifierWorkspaceReachabilityAnalysisIntentResource{}
+
+type ManagerVerifierWorkspaceReachabilityAnalysisIntentResource struct{}
+
+func (ManagerVerifierWorkspaceReachabilityAnalysisIntentResource) IDValidationFunc() pluginsdk.SchemaValidateFunc {
+	return reachabilityanalysisintent.ValidateReachabilityAnalysisIntentID
+}
+
+func (ManagerVerifierWorkspaceReachabilityAnalysisIntentResource) ResourceType() string {
+	return "azurerm_network_manager_verifier_workspace_reachability_analysis_intent"
+}
+
+func (ManagerVerifierWorkspaceReachabilityAnalysisIntentResource) ModelObject() interface{} {
+	return &ManagerVerifierWorkspaceReachabilityAnalysisIntentResourceModel{}
+}
+
+type ManagerVerifierWorkspaceReachabilityAnalysisIntentResourceModel struct {
+	Description           string                                                        `tfschema:"description"`
+	DestinationResourceId string                                                        `tfschema:"destination_resource_id"`
+	IpTraffic             []ManagerVerifierWorkspaceReachabilityAnalysisIntentIpTraffic `tfschema:"ip_traffic"`
+	Name                  string                                                        `tfschema:"name"`
+	SourceResourceId      string                                                        `tfschema:"source_resource_id"`
+	VerifierWorkspaceId   string                                                        `tfschema:"verifier_workspace_id"`
+}
+
+type ManagerVerifierWorkspaceReachabilityAnalysisIntentIpTraffic struct {
+	DesticationIps   []string                                      `tfschema:"destination_ips"`
+	DesticationPorts []string                                      `tfschema:"destination_ports"`
```

Comment:
Could you fix this typo?

```suggestion
	DestinationIps   []string                                      `tfschema:"destination_ips"`
	DestinationPorts []string                                      `tfschema:"destination_ports"`
```

Reply:
fixed